### PR TITLE
tests/MM-T947 : When deactivating users in the System Console, email address should not disappear

### PR DESCRIPTION
**Pull request by M-ZubairAhmed**
#### Ticket Link

https://automation-test-cases.vercel.app/test/MM-T947
#### Related Pull Requests

- Has server changes (please link here)
- Has redux changes (please link here)
- Has mobile changes (please link here)

#### Release Note
```release-note
NONE
```

### CODE REVIEW

**Review by hmhealey — APPROVED** [review]

File: e2e/cypress/integration/system_console/user_management/users_deactivation_spec.js [hunk 1]
```diff
@@ -161,4 +170,46 @@ describe('System Console > User Management > Deactivation', () => {
...
+        // # Visit the system console.
+        cy.visit('/admin_console');
+
+        // # Go to User management / Users tab
+        cy.findByTestId('user_management.system_users').should('be.visible').click();
+
+        // # Create a new user
+        cy.apiCreateUser().then(({user: user1}) => {
+            // # Search the newly created user in the search box
+            cy.findByPlaceholderText('Search users').should('be.visible').clear().type(user1.email);
```
Comment (josephbaylon) [inline e2e/cypress/integration/system_console/user_management/users_deactivation_spec.js:184]: Please add a delay here to account for async DOM refresh. The test is failing for me locally without this delay.

```suggestion
            cy.findByPlaceholderText('Search users').should('be.visible').clear().type(user1.email).wait(TIMEOUTS.HALF_SEC);
```

**Review by josephbaylon — CHANGES_REQUESTED** [review]
Overall looks good. Just one minor delay for test stability.

**Review by josephbaylon — APPROVED** [review]
LGTM. Test passed locally. Thanks @M-ZubairAhmed !!

**Review by prapti — APPROVED** [review]
Thank you so much yet again @M-ZubairAhmed! This is great: the test does exactly what's expected and is very clean.

**Pull request merged**